### PR TITLE
feat: amido to ensono

### PR DESCRIPTION
**Pull request by isaac-shek**
#### 📲 What
Ensono company now

#### 🤔 Why
		
Why it's needed, background context.
		
#### 🛠 How
		
More in-depth discussion of the change or implementation.

#### 👀 Evidence
		
Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR.
		 
#### 🕵️ How to test

Notes on how a reviewer can test the changes, e.g. how to run the tests.

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -48,22 +48,22 @@ No problem! You can learn how from this *free* series, [How to Contribute to an
...
 2. If you've added code that should be tested, add tests.
 3. Update the documentation.
 4. Ensure the test suite passes.
-5. Issue the pull request using the repos PULL_REQUEST_TEMPLATE.md. It's important to link the [issue number](https://github.com/amido/amido.github.io/issues) to ensure that the Issue is closed.
+5. Issue the pull request using the repos PULL_REQUEST_TEMPLATE.md. It's important to link the [issue number](https://github.com/Ensono/amido.github.io/issues) to ensure that the Issue is closed.
 
 ## How to report a bug
 ### Security first
-If you find a security vulnerability, do NOT open an [issue](https://github.com/amido/amido.github.io/issues/new/choose). Email [Amido](mailto:stacks@amido.com) instead.
+If you find a security vulnerability, do NOT open an [issue](https://github.com/Ensono/amido.github.io/issues/new/choose). Email [Ensono](mailto:stacks@amido.com) instead.
```
Comment (leesaxby) [inline .github/CONTRIBUTING.md:66]: Do we have an ensono email for this `stacks@amido.com`?

Comment (isaac-shek) [reply .github/CONTRIBUTING.md]: Yes we do, updated.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -91,7 +91,7 @@ For versioning, we follow [Semantic Version 2.0.0](https://semver.org/).
 Code styling and practices depend on the project. Check the project repo for more information.
 
 # Community
-If you are interested in talking with Amido on how accelerate your cloud adoption using Amido Stacks then please reach out at [Amido](mailto:london@amido.com).
+If you are interested in talking with Ensono on how accelerate your cloud adoption using Ensono Stacks then please reach out at [Ensono](mailto:london@amido.com).
```
Comment (leesaxby) [inline .github/CONTRIBUTING.md:94]: Ensono emial here also? `mailto:london@amido.com`

Comment (isaac-shek) [reply .github/CONTRIBUTING.md]: Yes, good spot - updated.

**Review by georgewilde — APPROVED** [review]

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -1,25 +1,25 @@
...
 However, we have a few baselines worth noting:
 * Please don't expect your new feature to be implemented unless you or someone else sends a pull request. Submit with a test to get it merged quicker.
 * If you have an issue, please tell us how to reproduce it. Bonus points for a failing test case.
 * Not everything will be fixed.
 * Did we mention we love tests?
 
-Please, don't use the issue tracker if your problem is not strictly Amido Stacks specific. Stack Overflow or other sources are worth considering.
+Please, don't use the issue tracker if your problem is not strictly Ensono Stacks specific. Stack Overflow or other sources are worth considering.
 
 _See the individual [Amido projects](./README.md) for in depth information about environment setup, testing, and code styles._
```
Comment (leesaxby) [inline .github/CONTRIBUTING.md:24]: Does this one need changing? `[Amido projects]`

Comment (georgewilde) [reply .github/CONTRIBUTING.md:24]: Oh good spot that man

File: docs/assets/README.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ As per the [Docusaursus documentation](https://docusaurus.io/docs/en/doc-markdow
 
 To reduce the size of the monorepo, we instead insist on ALL assets to be hosted publically.
 
-## Amido Maintainers
+## Ensono Maintainers
 
 If you have access to the `Amido.Stacks` subscription in Azure, under the `amido-stacks-assets` resource group there is a Storage Account named `amidostacksassets`. This contains `docs` a _public read only storage container_ under which you may upload all static assets to the `assets` folder.
```
Comment (leesaxby) [inline docs/assets/README.md:9]: These ones?
`Amido.Stacks`
`amido-stacks-assets`
`amidostacksassets`

Comment (isaac-shek) [reply docs/assets/README.md:9]: we will wait for Russell to move the assets to Ensono blob storage

File: .github/CONTRIBUTING.md (hunk 3 above)
Comment (isaac-shek) [reply .github/CONTRIBUTING.md]: updated

**Review by leesaxby — APPROVED** [review]

**Pull request merged**